MSAL/src/MSALPublicClientApplication.m

```diff
@@ -1395,9 +1400,6 @@ - (BOOL)shouldExcludeValidationForAuthority:(MSIDAuthority *)authority
         for (MSALAuthority *knownAuthority in self.internalConfig.knownAuthorities)
         {
             if ([authority isKindOfClass:knownAuthority.msidAuthority.class]
```

Comment:
Unreleated to this PR. Can we remove the redundant if check for self.internalConfig.knowsAuthorities. The inner for loop should be sufficient to check whether the list is empty or nil.